README.md

```diff
@@ -2514,7 +2517,7 @@ Nat. Nanotechnol., 2023.
 ### Schlieren imaging
 - <a name="Shiba23tpami"></a>Shiba, S., Hamann, F., Aoki, Y., Gallego, G.,  
 *[Event-based Background-Oriented Schlieren](https://doi.org/10.1109/TPAMI.2023.3328188)*,  
-IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Project page](https://github.com/tub-rip/event_based_bos), [Video](https://youtu.be/v6ms6g2eOB8), [PDF](https://arxiv.org/pdf/2311.00434)
+IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Project page](https://github.com/tub-rip/event_based_bos), [Video (short, only results)](https://youtu.be/v6ms6g2eOB8), [Narrated Video (5min)](https://youtu.be/Ev52n8KgxIU?si=0_Jpy1p-cGulfBTx), [PDF](https://arxiv.org/pdf/2311.00434)
```

Comment:
I think only one video is needed: the best one, which is also HD.

Reply:
Fixed the video link to the latest one.